pbs/README.md

```diff
@@ -0,0 +1,57 @@
+# PBS
+
+Proposer builder seperation for Op Stack.
+
+## How to run the devnet
+
+You first need to build the op-geth docker image with builder API support.
+
+```shell
+$ git clone git@github.com:flashbots/op-geth.git
+$ cd op-geth
+$ docker build . -t <YOUR_OP_GETH_BUILDER_IMAGE>
+```
+
+Then you can run the devnet with the following command:
+
+```shell
+$ BUILDER_IMAGE=<YOUR_OP_GETH_BUILDER_IMAGE> DEVNET_BUILDER=true make devnet-up
+```
+
+To additionally enable load testing through [tx-fuzz](https://github.com/MariusVanDerWijden/tx-fuzz), you can run the following command:
+
+```shell
+$ BUILDER_IMAGE=<YOUR_OP_GETH_BUILDER_IMAGE> DEVNET_LOAD_TEST=true DEVNET_BUILDER=true make devnet-up
+```
+
+If the `BUILDER_IMAGE` is not set, the devnet will use the image from `flashbots/op-geth:latest`.
```

Comment:
this should be at the start of as the user doesn't need to build the docker image anymore to run the devnet